### DISCUSSION TITLE
shortcut delete row doesnt work when implementing react-select

### DISCUSSION THREAD

**Discussion by rifqishiddiq** [nick-keller/react-datasheet-grid#288] [Q&A] [answered]
Hi,
by pressing delete (4 times) on selected cell, the row will be deleted. but if implementing react-select, the row still intact and doesnt get deleted. Is it intended to be like that?

**Comment by nick-keller** [comment, marked as answer]
Have a look a [isCellEmpty](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#iscellempty), maybe this is the issue.
